### PR TITLE
[ARGG-784]: Release pipeline for cra5 testing

### PR DESCRIPTION
**Pull request by olliecurtis**
**Note: More changes were made in this PR than just simply adding a `yml` file because when the CI was running it was complaining that the linting checks were failing so they needed to be fixed possibly because the lockfile was pointing at artifactory so could never get the deps to properly check.**

As part of our work for testing CRA 5, we want to setup releases for each time we merge to the `fork_cra5` branch for dev releases so this PR adds this ability to the branch.

The idea behind this is for quick testing purposes when we merge into the branch an `alpha` tag version is released to npm for allowing us to test with changes.

Then for stable and betas we would still do that manually for now and then when it comes to it we would have CI that would push stables in fork later on!

### CODE REVIEW

**Comment by olliecurtis** [comment]
Not 100% sure why linting is complaining on files I have not touched right now. Keeping an eye out for it though

File: package-lock.json
Comment (olliecurtis) [inline package-lock.json:1]: I updated this as it had all references to our internal artifactory so couldn't pull any dependencies to allow CI to work

File: packages/react-scripts/backpack-addons/README.md [hunk 1]
```diff
@@ -2,27 +2,27 @@ Our react scripts fork includes a number of custom configuration items in order
...
+| **crossOriginLoading**      | Option to enable cross origin loading of chunks to modify the default webpack behaviour of being false. <br> Docs: https://webpack.js.org/configuration/output/#outputcrossoriginloading                                                                                                                                                                                                                                                      | **false**                                                                                                                                       |
```
[the remark this answers is not in the document]
Comment (olliecurtis) [reply packages/react-scripts/backpack-addons/README.md:9]: This was all coming from prettier formatting, as our CI was failing for it, running the provided commands produced these results

[the remark this answers is not in the document]
Comment (soulcheck) [reply packages/react-scripts/backpack-addons/README.md:9]: Makes sense

**Review by soulcheck — APPROVED** [review]
LGTM

Looks like setup-node supports package caching [out of the box](https://github.com/actions/setup-node#caching-global-packages-data), but if that works would require a separate investigation.

**Pull request merged**